main.js

```diff
@@ -0,0 +1,321 @@
+        {keyCode: 221, shifted_ru: 'Ъ', shifted_en: '}', en: ']', ru: 'ъ', width:1 },
+        {keyCode: 220, shifted_ru: '/', shifted_en: '|', en: '\\', ru: '\\', width:1 },
+        {keyCode: 46, title: 'Del', width:1 },
+    ],
+    [
+        {keyCode: 20, title: 'CapsLock', width:2 },
+        {keyCode: 65, shifted_ru: 'Ф', shifted_en: 'A', en: 'a', ru: 'ф', width:1 },
+        {keyCode: 83, shifted_ru: 'Ы', shifted_en: 'S', en: 's', ru: 'ы', width:1 },
+        {keyCode: 68, shifted_ru: 'В', shifted_en: 'D', en: 'd', ru: 'в', width:1 },
+        {keyCode: 70, shifted_ru: 'А', shifted_en: 'F', en: 'f', ru: 'а', width:1 },
+        {keyCode: 71, shifted_ru: 'П', shifted_en: 'G', en: 'g', ru: 'п', width:1 },
+        {keyCode: 72, shifted_ru: 'Р', shifted_en: 'H', en: 'h', ru: 'р', width:1 },
+        {keyCode: 74, shifted_ru: 'О', shifted_en: 'J', en: 'j', ru: 'о', width:1 },
+        {keyCode: 75, shifted_ru: 'Л', shifted_en: 'K', en: 'k', ru: 'л', width:1 },
+        {keyCode: 76, shifted_ru: 'Д', shifted_en: 'L', en: 'l', ru: 'д', width:1 },
+        {keyCode: 59, shifted_ru: 'Ж', shifted_en: ':', en: ';', ru: 'ж', width:1 },
+        {keyCode: 222, shifted_ru: 'Э', shifted_en: '"', en: "'", ru: 'э', width:1 },
+        {keyCode: 13, value:"\n", title: 'Enter', width:2 },
+    ],
+    [
+        {keyCode: 16, code: 'ShiftLeft', title: 'Shift', width:2 },
+        {keyCode: 90, shifted_ru: 'Я', shifted_en: 'Z', en: 'z', ru: 'я', width:1 },
+        {keyCode: 88, shifted_ru: 'Ч', shifted_en: 'X', en: 'x', ru: 'ч', width:1 },
+        {keyCode: 67, shifted_ru: 'С', shifted_en: 'C', en: 'c', ru: 'с', width:1 },
+        {keyCode: 86, shifted_ru: 'М', shifted_en: 'V', en: 'v', ru: 'м', width:1 },
+        {keyCode: 66, shifted_ru: 'И', shifted_en: 'B', en: 'b', ru: 'и', width:1 },
+        {keyCode: 78, shifted_ru: 'Т', shifted_en: 'N', en: 'n', ru: 'т', width:1 },
+        {keyCode: 77, shifted_ru: 'Ь', shifted_en: 'M', en: 'm', ru: 'ь', width:1 },
+        {keyCode: 188, shifted_ru: 'Б', shifted_en: '<', en: ',', ru: 'б', width:1 },
+        {keyCode: 190, shifted_ru: 'Ю', shifted_en: '>', en: '.', ru: 'ю', width:1 },
+        {keyCode: 191, shifted_ru: ',', shifted_en: '?', en: '/', ru: '.', width:1 },
+        {keyCode: 38, title: '&uarr;', width:1 },
+        {keyCode: 16, code: 'ShiftRight', title: 'Shift', width:2 },
+    ],
+    [
+        {keyCode: 17, code: 'ControlLeft', title: 'Ctrl', width:1 },
+        {keyCode: 91, title: 'Win', width:1 },
+        {keyCode: 18, code: 'AltLeft', title: 'Alt', width:1 },
+        {keyCode: 32, code: 'Space', title: ' ', width:7 },
+        {keyCode: 18, code: 'AltRight', title: 'Alt', width:1 },
+        {keyCode: 37, title: '&larr;', width:1 },
+        {keyCode: 40, title: '&darr;', width:1 },
+        {keyCode: 39, title: '&rarr;', width:1 },
+        {keyCode: 17, code: 'ControlRight', title: 'Ctrl', width:1 },
+    ]
+]
+let container = document.createElement('div');
```

Comment:
it is better to have separate funcion for initial render

Reply:
Used separate funcion.